internal/config/config.go

```diff
@@ -41,8 +41,8 @@ func Read(config *Configuration, configFile string) (found bool, configFileUsed
 	v.AddConfigPath(".")
 	v.AddConfigPath("$HOME/.config/")
 	v.AddConfigPath("/etc")
-	v.SetConfigName("preprocessing_moma")
-	v.SetEnvPrefix("PREPROCESSING_MOMA")
+	v.SetConfigName("preprocessing")
+	v.SetEnvPrefix("PREPROCESSING")
```

Comment:
Maybe "SDPS_PREPROCESSING" or "ENDURO_PREPROCESSING" would be better to link preprocessing to Enduro and provide more distinct namespacing?

Reply:
Cool, I agree, ENDURO_PREPROCESSING looks good to me.